lib/src/code_parser.dart

```diff
@@ -12,6 +12,7 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
+import 'dart:io';
 // ignore: implementation_imports
```

Comment:
(Unrelated, but this ignore is probably unnecessary?)

Reply:
I had kept it because it's unrelated ;-) Anyhow, removed that line now.